server/chat-plugins/trivia.ts

```diff
@@ -686,12 +699,16 @@ class Trivia extends Rooms.RoomGame {
 		this.room.roomlog(buf);
 		this.room.modlog(`(${this.room.roomid}) ${logbuf}`);
 
+		if (!triviaData.history) triviaData.history = [];
+		triviaData.history?.push(this.game);
+		if (triviaData.history && triviaData.history.length > 10) triviaData.history = triviaData.history.slice(-10);
```

Comment:
```suggestion
		if (!triviaData.history) triviaData.history = [];
		triviaData.history.push(this.game);
		if (triviaData.history.length > 10) triviaData.history.unshift();
```

After guaranteeing `triviaData.history` exists; you don't need to re-check it every line.